Sources/LocheckCommand/main.swift

```diff
@@ -280,6 +283,12 @@ struct DiscoverLproj: ParsableCommand {
             print("Translations to check: \(translationLproj.count)")
```

Comment:
Same, consider not printing this if there are no translations

Reply:
I think this one is OK, I'll change the other one.